Sources/Widgets/Representations/WidgetRepresentation/index.d.ts

```diff
@@ -69,6 +70,22 @@ export function newInstance(initialValues?: IWidgetRepresentationInitialValues):
  */
 export function getPixelWorldHeightAtCoord(worldCoord: number[], displayScaleParams: IDisplayScaleParams): number[];
 
+export interface IPipeline {
```

Comment:
Maybe call this `IWidgetPipeline`? It seems specific enough, while `IPipeline` sounds a bit broadly scoped, even though it's exported from WidgetRepresentation.